jaeger_client/span.py

```diff
@@ -92,6 +89,19 @@ def set_tag(self, key, value):
                 self.tags.append(tag)
         return self
 
+    def _set_sampling_priority(self, value):
```

Comment:
joe made a good point in the node PR; if the span is already debug sampled, setting the tag again should NOT call is_allowed as that can decrement the credits and leave the client and the agent out of sync

Reply:
Definitely. We should probably update the Go client too at some point with this fix.